pretenders/server/utils.py

```diff
@@ -1,11 +1,5 @@
 import bottle
 
-try:
-    from collections import OrderedDict
-except ImportError:
-    #2.6 compatibility
-    from pretenders.common.compat.ordered_dict import OrderedDict
-
```

Comment:
BTW if we don't care about `OrderedDict` or 2.6 compatibility, we may as well get rid of that module...